pyclient/docs/source/examples.rst

```diff
@@ -104,29 +128,36 @@ Join 2 tables
     session = Session()
 
     table1 = session.empty_table(5)
+
     table1 = table1.update(["Values1 = i", "Group = i"])
+
     table2 = session.empty_table(5)
+
     table2 = table2.update(["Values2 = i + 10", "Group = i"])
 
     table = table1.join(table2, on=["Group"])
+
     session.bind_table(name="MyTable", table=table)
 
 Use a combo aggregation on a table
 ##################################
 
-Combined aggregations can be executed on tables in the Python client. This example creates a combo aggregation that averages the `Count` column of a table, and aggregates it by the `Group` column::
+Combined aggregations can be executed on tables in the Python client. This example creates a combo aggregation that averages the `Count` column of a table, and aggregates it by the `Group` column:
```

Comment:
needs an update after the refactor?

Reply:
I don't think the refactor affected the pyclient package